src/Nerve_GIC/include/gudhi/GIC.h

```diff
@@ -464,7 +444,7 @@ class Cover_complex {
         std::vector<int> samples(m);
         SampleWithoutReplacement(n, m, samples);
```

Comment:
It does not compile with TBB
```suggestion
        SampleWithoutReplacement(this->num_points, m, samples);
```

Reply:
Ça doit venir de cette fonction je pense :

double GetUniform() {
    thread_local std::default_random_engine re;
    std::uniform_real_distribution<double> Dist(0, 1);
    return Dist(re);
  }

mais je t'avoue je ne sais pas trop comment fix ça

Reply:
aah ok sorry I thought the problem was coming from parallel computations. ok that's easy to fix then ;-)